tests/scripts/basic-build-test.sh

```diff
@@ -217,3 +271,5 @@ make clean
 if [ -f "$CONFIG_BAK" ]; then
     mv "$CONFIG_BAK" "$CONFIG_H"
 fi
+
+exit $EXITCODE
```

Comment:
`$EXITCODE` should also indicate failure if any test fails - not just if the test coverage is below the set threshold.

Reply:
Done!